src/AppBundle/EventListener/RestInputOuputFormatter.php

```diff
@@ -111,6 +123,20 @@ private function arrayToResponse($data, Request $request)
         return $response;
     }
 
+    private function containsEntity($object)
+    {
+        if (is_array($object)) {
+            foreach($object as $subObject) {
+                if ($this->containsEntity($subObject)) {
+                    return true;
+                }
+            }
+            return false;
+        }
+
+        return is_object($object) && strpos(get_class($object), 'Entity') !== false;
```

Comment:
Is there not an interface we can use here? If we're using strpos we may as well var_export the whole object and call strpos(), and remove the loop